Tools/ADT/auto_cl_on_merge.py

```
@@ -0,0 +1,146 @@
+
+def main():
+    if len(sys.argv) < 4:
+        logging.error("Usage: auto_cl_on_merge.py <PR_NUMBER> <PR_BODY_FILE> <PR_AUTHOR>")
+        sys.exit(1)
+
+    pr_number = sys.argv[1]
+    body_file = Path(sys.argv[2])
+    default_author = sys.argv[3]
+
+    if not body_file.exists():
+        logging.error(f"Файл с телом PR не найден: {body_file}")
```

Ruff (0.16.1)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)